imports/ui/pages/professor/questions_library.jsx

```diff
@@ -173,9 +174,9 @@ class _QuestionsLibrary extends Component {
   }
 }
 
-export const QuestionsLibrary = createContainer(() => {
+export const QuestionsLibrary = createContainer(props => {
   const handle = Meteor.subscribe('courses') && Meteor.subscribe('questions.library')
```

Comment:
Here, I think we might be able to not subscribe to courses anymore (unless we need it to know if the user is an instructor).

I think we want to update the questions.library publication to take courseId as an argument, which means we don't have to query for the courseId later.  This should make the subscription to question library faster. This also means that we might not need to pass a courseId prop to the questionSidebar, if Sidebar is just diplaying the questions that are passed as a prop. 

 We should also consider whether to add courseId as an index for the Questions Collection in Mongo. - not sure about this last point, it depends how often this subscription is called; we mainly want to index based on those subscriptions made during the live sessions.